ffac-wg-registration/files/lib/gluon/wg-registration/registration.sh

```diff
@@ -4,8 +4,11 @@ if [ "$(uci get gluon.mesh_vpn.enabled)" = "true" ] || [ "$(uci get gluon.mesh_v
 	# check if registration has been done since last boot
 	if [ ! -f /tmp/WG_REGISTRATION_SUCCESSFUL ]; then
 		# Push public key to broker, test for https and use if supported
-		wget -q "https://[::1]"
-		if [ $? -eq 1 ]; then
+		ret=0
+		wget -q "https://[::1]" 2>/dev/null || ret=$?
```

Comment:
Why do you introduce the `ret` instead of having the original code?

Reply:
To make this passage more similar. We can of course also switch to the other way around or keep the difference.
Whatever you like better :)

Reply:
The `|| ret=` is necessary if the script uses `set -e` and otherwise the old code is fine. 

Just wondering if this additional complexity is worth it?